build.gradle

```diff
@@ -16,9 +16,9 @@ buildscript {
 
 allprojects {
     repositories {
+        jcenter()
```

Comment:
Thanks for working on cleanup. Removing unused imports makes sense but does this change in build.gradle make any difference?

Reply:
Do you mean you removed [the 'Update build.gradle' commit](https://github.com/commons-app/apps-android-commons/pull/1606/commits/259f8c05d7097496711915add7a4acbe2841fe3d) already locally? If so, please amend and update your public branch.

Reply:
@dlsrks1021 any update?

Reply:
I removed 'Update build.gradle' commit and modify 'remove unused imports' commit to do not change build.gradle.

Reply:
Thanks! :+1: